pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -217,14 +217,64 @@ rec {
       '';
 
       meta = with lib; {
-        description = "Color and Black-and-White emoji fonts";
+        description = "Color emoji font";
         homepage = "https://github.com/googlefonts/noto-emoji";
         license = with licenses; [ ofl asl20 ];
         platforms = platforms.all;
         maintainers = with maintainers; [ mathnerd314 sternenseemann ];
       };
     };
 
+  noto-fonts-monochrome-emoji =
+    # Metadata fetched from
+    #  https://www.googleapis.com/webfonts/v1/webfonts?key=${GOOGLE_FONTS_TOKEN}&family=Noto+Emoji
+    let metadata = with builtins; head (fromJSON (readFile ./noto-emoji.json)).items;
+
+    in
+    stdenvNoCC.mkDerivation rec {
+      pname = "noto-fonts-monochrome-emoji";
+      version = "unstable-${metadata.lastModified}";
+      preferLocalBuild = true;
+
+      dontUnpack = true;
+      srcs = with lib; let
```

Comment:
- `with lib;` -> `lib.mapAttrsToList` below
- could you lift this out of the derivation to line 231?
- I think the manual alignment will have to go, even though it looks nice
- It may be better transposed, but feel free to ignore:
```nix
{
   "300" = {
        weight = "Light";
        hash = "sha256-AXGLdWebddyJhTKMW/D/6tW8ODcaXrUM96m2hN9wYlg=";
    };
}
````